examples/tox21/tox21_tensorgraph_graph_conv.py

```diff
@@ -6,10 +6,14 @@
 from __future__ import unicode_literals
```

Comment:
Couple general thoughts about this file:

1. First, amazing work @lilleswing! It's very cool that sophisticated models like these graph-convs can now be built and trained entirely in user-space. This should make algorithmic development in DeepChem much more feasible.
2. Because we've exposed the internals, this code looks messier than our old graph-conv examples. While those did mysterious black-magic under the hood, they had the advantage of looking stylistically straight-forward. Let's brainstorm ways to increase simplicity while keeping algorithms in user-space.